packer_templates/rocky/http/8/ks.cfg

```diff
@@ -0,0 +1,77 @@
+install
```

Comment:
Is there any reason not to just use the centos 8 ks file here so we avoid the need to manage another file?

Reply:
Actually it's the same, left here just to have all files in place w/o additional actions.